mkdocs.yml

```diff
@@ -95,8 +96,8 @@ markdown_extensions:
   # pymdownx.superfences is deprecated.
   # It must be replaced with pymdownx.tabbed but it requires new CSS
   # TODO with material 5 update
-  - pymdownx.superfences:
-      legacy_tab_classes: true
+  ### - pymdownx.superfences:
+  ###    legacy_tab_classes: true
```

Comment:
I have no clue how to make `mkdocs serve` work with this option.

Reply:
Did you try to run it in a Python virtual environment and run `pip install -r common/build_tools/requirements.txt` ?

Reply:
yes

```
~/.opt/byz-f/doc.teku q9-teku-testnet
❯ pip install -r common/build_tools/requirements.txt
Defaulting to user installation because normal site-packages is not writeable
Requirement already satisfied: mkdocs==1.1 in /home/user/.local/lib/python3.8/site-packages (from -r common/build_tools/requirements.txt (line 1)) (1.1)
Requirement already satisfied: mkdocs-material==4.6.3 in /home/user/.local/lib/python3.8/site-packages (from -r common/build_tools/requirements.txt (line 2)) (4.6.3)
Requirement already satisfied: Markdown==3.2.1 in /home/user/.local/lib/python3.8/site-packages (from -r common/build_tools/requirements.txt (line 3)) (3.2.1)
Requirement already satisfied: markdown-fenced-code-tabs==1.0.3 in /home/user/.local/lib/python3.8/site-packages (from -r common/build_tools/requirements.txt (line 4)) (1.0.3)
Requirement already satisfied: markdown-include==0.5.1 in /home/user/.local/lib/python3.8/site-packages (from -r common/build_tools/requirements.txt (line 5)) (0.5.1)
Requirement already satisfied: MarkupSafe==1.1.1 in /usr/lib/python3.8/site-packages (from -r common/build_tools/requirements.txt (line 6)) (1.1.1)
Requirement already satisfied: mkdocs-markdownextradata-plugin==0.1.3 in /home/user/.local/lib/python3.8/site-packages (from -r common/build_tools/requirements.txt (line 7)) (0.1.3)
Requirement already satisfied: mkdocs-exclude==1.0.2 in /home/user/.local/lib/python3.8/site-packages (from -r common/build_tools/requirements.txt (line 8)) (1.0.2)
Requirement already satisfied: mkdocs-redirects==1.0.0 in /home/user/.local/lib/python3.8/site-packages (from -r common/build_tools/requirements.txt (line 9)) (1.0.0)
Requirement already satisfied: mkdocs-git-revision-date-localized-plugin==0.5.0 in /home/user/.local/lib/python3.8/site-packages (from -r common/build_tools/requirements.txt (line 10)) (0.5.0)
Requirement already satisfied: mkdocs-minify-plugin==0.3.0 in /home/user/.local/lib/python3.8/site-packages (from -r common/build_tools/requirements.txt (line 11)) (0.3.0)
Requirement already satisfied: livereload>=2.5.1 in /usr/lib/python3.8/site-packages (from mkdocs==1.1->-r common/build_tools/requirements.txt (line 1)) (2.6.1)
Requirement already satisfied: click>=3.3 in /usr/lib/python3.8/site-packages (from mkdocs==1.1->-r common/build_tools/requirements.txt (line 1)) (7.1.1)
Requirement already satisfied: Jinja2>=2.10.1 in /usr/lib/python3.8/site-packages (from mkdocs==1.1->-r common/build_tools/requirements.txt (line 1)) (2.11.2)
Requirement already satisfied: lunr[languages]==0.5.6 in /home/user/.local/lib/python3.8/site-packages (from mkdocs==1.1->-r common/build_tools/requirements.txt (line 1)) (0.5.6)
Requirement already satisfied: tornado>=5.0 in /usr/lib/python3.8/site-packages (from mkdocs==1.1->-r common/build_tools/requirements.txt (line 1)) (6.0.4)
Requirement already satisfied: PyYAML>=3.10 in /usr/lib/python3.8/site-packages (from mkdocs==1.1->-r common/build_tools/requirements.txt (line 1)) (5.3.1)
Requirement already satisfied: Pygments>=2.4 in /usr/lib/python3.8/site-packages (from mkdocs-material==4.6.3->-r common/build_tools/requirements.txt (line 2)) (2.6.1)
Requirement already satisfied: pymdown-extensions>=6.3 in /usr/lib/python3.8/site-packages (from mkdocs-material==4.6.3->-r common/build_tools/requirements.txt (line 2)) (6.3)
Requirement already satisfied: setuptools>=36 in /usr/lib/python3.8/site-packages (from Markdown==3.2.1->-r common/build_tools/requirements.txt (line 3)) (46.1.3)
Requirement already satisfied: htmlmin>=0.1.12 in /usr/lib/python3.8/site-packages (from markdown-fenced-code-tabs==1.0.3->-r common/build_tools/requirements.txt (line 4)) (0.1.12)
Requirement already satisfied: babel>=2.7.0 in /home/user/.local/lib/python3.8/site-packages (from mkdocs-git-revision-date-localized-plugin==0.5.0->-r common/build_tools/requirements.txt (line 10)) (2.8.0)
Requirement already satisfied: GitPython in /home/user/.local/lib/python3.8/site-packages (from mkdocs-git-revision-date-localized-plugin==0.5.0->-r common/build_tools/requirements.txt (line 10)) (3.1.2)
Requirement already satisfied: jsmin>=2.2.2 in /usr/lib/python3.8/site-packages (from mkdocs-minify-plugin==0.3.0->-r common/build_tools/requirements.txt (line 11)) (2.2.2)
Requirement already satisfied: six in /usr/lib/python3.8/site-packages (from livereload>=2.5.1->mkdocs==1.1->-r common/build_tools/requirements.txt (line 1)) (1.14.0)
Requirement already satisfied: future>=0.16.0 in /home/user/.local/lib/python3.8/site-packages (from lunr[languages]==0.5.6->mkdocs==1.1->-r common/build_tools/requirements.txt (line 1)) (0.18.2)
Requirement already satisfied: nltk>=3.2.5; extra == "languages" in /home/user/.local/lib/python3.8/site-packages (from lunr[languages]==0.5.6->mkdocs==1.1->-r common/build_tools/requirements.txt (line 1)) (3.5)
Requirement already satisfied: pytz>=2015.7 in /home/user/.local/lib/python3.8/site-packages (from babel>=2.7.0->mkdocs-git-revision-date-localized-plugin==0.5.0->-r common/build_tools/requirements.txt (line 10)) (2020.1)
Requirement already satisfied: gitdb<5,>=4.0.1 in /home/user/.local/lib/python3.8/site-packages (from GitPython->mkdocs-git-revision-date-localized-plugin==0.5.0->-r common/build_tools/requirements.txt (line 10)) (4.0.5)
Requirement already satisfied: joblib in /home/user/.local/lib/python3.8/site-packages (from nltk>=3.2.5; extra == "languages"->lunr[languages]==0.5.6->mkdocs==1.1->-r common/build_tools/requirements.txt (line 1)) (0.14.1)
Requirement already satisfied: regex in /home/user/.local/lib/python3.8/site-packages (from nltk>=3.2.5; extra == "languages"->lunr[languages]==0.5.6->mkdocs==1.1->-r common/build_tools/requirements.txt (line 1)) (2020.4.4)
Requirement already satisfied: tqdm in /home/user/.local/lib/python3.8/site-packages (from nltk>=3.2.5; extra == "languages"->lunr[languages]==0.5.6->mkdocs==1.1->-r common/build_tools/requirements.txt (line 1)) (4.46.0)
Requirement already satisfied: smmap<4,>=3.0.1 in /home/user/.local/lib/python3.8/site-packages (from gitdb<5,>=4.0.1->GitPython->mkdocs-git-revision-date-localized-plugin==0.5.0->-r common/build_tools/requirements.txt (line 10)) (3.0.4)

~/.opt/byz-f/doc.teku q9-teku-testnet*
❯ mkdocs serve                                      
INFO    -  Building documentation... 
ERROR   -  Config value: 'markdown_extensions'. Error: legacy_tab_classes 

Aborted with 1 Configuration Errors!
```